WalletWasabi/Wallets/IWallet.cs

```diff
@@ -11,6 +11,7 @@ namespace WalletWasabi.Wallets;
 public interface IWallet
 {
 	string WalletName { get; }
+	Guid Id { get; }
```

Comment:
I am not sure about this but I would like to see WalletId type instead of Guid everywhere. More readable and we know what we are looking at. Guid is not descriptive. WalletId would be a simple Guid wrapper class. 

@kiminuo WDYT?

Reply:
>  one probably also needs a JSON converter to serialize/deserialize that value in JSON.

Little addition to this, we are not planning to serialize it, but always create the GUID when the instance is created.

Reply:
Done. Please, tell me what you think.